src/index.test.ts

```diff
@@ -1407,3 +1409,28 @@ test('Should be able to configure UnleashClient with a URL instance', () => {
     const client = new UnleashClient(config);
     expect(client).toHaveProperty('url', url);
 });
+
+test('Should be immediately ready when bootstrapping', () => {
```

Comment:
this test is passing for me even before your changes so I'm wondering what is this change about?